install/ui/src/freeipa/user.js

```diff
@@ -218,6 +218,15 @@ return {
                             label: '@i18n:objects.cert.certificates'
                         },
                         {
+                            $type: 'certmap_multivalued',
```

Comment:
Would be good if it had the question mark hint to describe what is the purpose of this field. What problem can be solved by it. I believe it is not self explanatory.